include/battle.h

```diff
@@ -117,6 +117,7 @@ struct ProtectStruct
     u32 spikyShielded:1;
     u32 kingsShielded:1;
     u32 banefulBunkered:1;
+    u32 obstruct:1;
```

Comment:
Maybe
```suggestion
    u32 obstructed:1;
```
to match the other protecting move flag names.